client/src/plugins/version-info/VersionInfo.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React, {
+  Fragment,
+  PureComponent
+} from 'react';
+
+import classNames from 'classnames';
+
+import { Fill } from '../../app/slot-fill';
+import Metadata from '../../util/Metadata';
+import { VersionInfoOverlay } from './VersionInfoOverlay';
+
+import css from './VersionInfo.less';
+
+
+const CONFIG_KEY = 'versionInfo';
+
+export class VersionInfo extends PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    this._version = Metadata.version;
+    if (process.env.NODE_ENV === 'test') {
```

Comment:
Why don't we rely on the version provided by Metadata.version during TEST, too?

Reply:
No reason. I'll remove that.